py/tests/test_auth.py

```diff
@@ -2,6 +2,8 @@
 import sentry_relay
 import pytest
 
+# flake8: noqa
```

Comment:
Why do we exempt the entire file from linting? Is this about line length? I'd rather bump the global line length to 100 instead of doing this.

Reply:
Actually, this was just a leftover from an earlier iteration. E501 is disabled since black enforces it already.